libs/sdk-ui-kit/src/Header/UpsellButton.tsx

```diff
@@ -0,0 +1,21 @@
+// (C) 2021 GoodData Corporation
+import React from "react";
+
+/**
+ * @internal
+ */
+export interface IUpsellButtonProps {
+    onUpsellButtonClick?: (e: React.MouseEvent) => void;
+}
+
+/**
+ * @internal
+ */
+export const UpsellButton: React.FC<IUpsellButtonProps> = ({ onUpsellButtonClick }) => {
+    return (
+        <button className="gd-button-small gd-button-primary gd-upsell-button" onClick={onUpsellButtonClick}>
+            <i className="gd-icon-star" />
+            <span className="gd-upgrade-button-text">Upgrade</span>
```

Comment:
Please support localization for `Upgrade`.

Reply:
I created formatmessage id for it thanks